store/landmarks.js

```diff
@@ -0,0 +1,39 @@
+
+//Action Constants
+const GOT_LANDMARKS = "GOT_LANDMARKS"
+const THREW_ERROR = "THREW_ERROR"
+
+//Action Creators
+const gotLandmarks = landmarks => ({type: GOT_LANDMARKS, landmarks})
+const threwError = err => ({type: THREW_ERROR, err})
+
+//Thunks
+export const fetchLandmarks = (db) => async dispatch => {
+  try {
+      db.once("value", snap => {
+        const data = [];
+        snap.forEach(child => {
+          const childObj = child.toJSON();
+          data.push({ ...childObj });
+        });
+        dispatch(gotLandmarks(data))
+      });
+  } catch (error) {
+    console.log(error)
+    dispatch(threwError(error))
+  }
+}
+
+
+//Sub-Reducer
+const initialState = {}
```

Comment:
can we initialize the state with the things we know will be in it? in this case it looks like it would be 
```{
data: {},
err: false
}```